sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Azure.Monitor.OpenTelemetry.Exporter.csproj

```diff
@@ -9,7 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="System.Text.Json" />
-    <PackageReference Include="OpenTelemetry" />
+    <PackageReference Include="OpenTelemetry" VersionOverride="0.8.0-beta.1"/>
```

Comment:
Why do you need the override? https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Packages.Data.props#L98 points to the same version.

Reply:
That doesn't work because of the `$(MSBuildProjectName.StartsWith('Microsoft.'))`

https://github.com/Azure/azure-sdk-for-net/blob/09ee616ad234d55f1a70db56d05b7815f99a833d/eng/Packages.Data.props#L96-L98

Currently, we can't update the version of OpenTelemetry because the Exporter took a dependency on v0.8  and there are a few breaking changes in v1.0.
My next set of changes are to fix all the breaking changes and then remove this Override.